### PR TITLE
feat(calls): Add appconfig and user preference for default setting of…

### PR DESCRIPTION
**Pull request by nickvergessen**
… media

### ☑️ Resolves

* Fix API side of #13446 

### ⚙️ Settings API

The endpoint for actually setting it is in the provisioning API:
if you have ocs_api_viewer installed: `/index.php/apps/ocs_api_viewer/view/provisioning_api#/operations/preferences-set-preference`
grafik

Sample:
```
POST /ocs/v2.php/apps/provisioning_api/api/v1/config/users/{appId}/{configKey}
{"configValue":"yes|no"}

POST /ocs/v2.php/apps/provisioning_api/api/v1/config/users/spreed/play_sounds
{"configValue":"no"}
```

### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included or not possible
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

File: lib/Config.php [hunk 1]
```diff
@@ -662,4 +663,18 @@ public function getGridVideosLimit(): int {
...
 	}
+
+	/**
+	 * User setting falling back to admin defined app config
+	 *
+	 * @param ?string $userId
+	 * @return bool
+	 */
+	public function getCallsStartWithoutMedia(?string $userId): bool {
+		if ($userId !== null && $this->config->getUserValue($userId, 'spreed', UserPreference::CALLS_START_WITHOUT_MEDIA) === 'no') {
```
Comment (Antreesy) [inline lib/Config.php:674]: That's supposed to evaluate to `true` here?
image
image

**Review by Antreesy — COMMENTED** [review]
Seems fine

File: lib/Config.php (hunk 1 above)
Comment (nickvergessen) [reply lib/Config.php]: Ops, only `no` was respected. fixed now

**Review by Antreesy — APPROVED** [review]
Tested
image

**Review by DorraJaouad — APPROVED** [review]
Tested

**Pull request merged**